modular_skyrat/modules/sec_haul/code/guns/guns.dm

```diff
@@ -925,10 +866,50 @@
 	ammo_type = /obj/item/ammo_casing/caseless/b473/hp
 	round_type = AMMO_TYPE_HOLLOWPOINT
 
-/obj/item/ammo_box/magazine/multi_sprite/g11/rubber
-	ammo_type = /obj/item/ammo_casing/caseless/b473/rubber
-	round_type = AMMO_TYPE_RUBBER
-
 /obj/item/ammo_box/magazine/multi_sprite/g11/ihdf
 	ammo_type = /obj/item/ammo_casing/caseless/b473/ihdf
 	round_type = AMMO_TYPE_IHDF
+
+/////////////////////////////////////////////////
+//////////////////SHOTGUNS///////////////////////
+/////////////////////////////////////////////////
+
+/obj/item/gun/ballistic/shotgun/m23
+	name = "\improper Armadyne Model 23-37"
+	desc = "A classic take on something that was also a classic. Classic. 8-shot."
+	icon_state = "riotshotgun"
+	inhand_icon_state = "shotgun"
+	mag_type = /obj/item/ammo_box/magazine/internal/shot/m23
+	w_class = WEIGHT_CLASS_HUGE
+	slot_flags = ITEM_SLOT_BACK|ITEM_SLOT_BELT|ITEM_SLOT_SUITSTORE
```

Comment:
belt slot shotguns are objectionable. 

Suit slot means they can be carried on literally any suit clothing instead of just the ones designed for weapons.

Reply:
I'm probably going to switch shotguns out wirh something else yeah. Though more shotguns is good.